Resources/EditorData/AtomicEditor/editor/ui/renameasset.tb.txt

```diff
@@ -6,10 +6,9 @@ TBLayout: axis: y, distribution: gravity, position: left
 	TBSeparator: gravity: left right, skin: AESeparator
 	TBLayout: gravity: left right, distribution-position: right bottom
 		TBTextField: text: "New Name:"
-		TBEditField: id: new_name, autofocus: 1
+		TBEditField: id: new_name
```

Comment:
The only dialogs where this actually makes a difference is in snap settings, rename resource, and create project.  All the others don't populate the first field so auto select doesn't do anything.  If you want, I can back out the others and just leave these 3